docs/concepts/search/searchable-properties/session-replay.mdx

```diff
@@ -244,6 +244,10 @@ The reason a replay was triggered. For example, `session` when `replaysSessionSa
 
 - **Type:** string
 
+### `screen`
+
+A specific Screen or View that the user visited during the replay. You can also search for multiple Screens at once using `screens` and passing in an array of strings. Alias of `url`, suited for mobile.
```

Comment:
Both
"Alias of `url`, suited for mobile."
"Alias of `screen`, suited for web."

confused me upon reading it, since the 'suited for...' is referring to the searchable property heading, rather than what is mentioned in the sentence. I'd drop the 'suited for...' part of this sentence. 🤔